main/migrations/test/20200826081934-InitFreetownOrganizationData.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
```

Comment:
This is for seed data for testing. 
db-migrate have scope feature: https://db-migrate.readthedocs.io/en/latest/Getting%20Started/commands/#scoping

so I use this to seed data to DB:
db-migrate up:test 

If we run `db-migrate up` as usually, it would not run mirgation under /test/ folder